### PR TITLE
Addressing ADF regression tests so values not present in old files are not parsed

### PR DESCRIPTION
**Pull request by amandadumi**
Fixing old tests for remaining ADF regression test errors in #673

### CODE REVIEW

**Comment by berquist** [comment]
Confirmation that these fixes work using a regression output diff:
```diff
10,40c10
<   /home/eric/development/cclib_berquist/test/../data/regression/ADF/ADF2004.01/dvb_gopt.adfout ... 3 test(s) had errors
< Error for testgeovalues_scfvalues (test.data.testGeoOpt.ADFGeoOptTest)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 160, in testgeovalues_scfvalues
<     count_scfvalues = len(self.data.scfvalues) - self.extrascfs
< AttributeError: 'ccData' object has no attribute 'scfvalues'
< 
< Error for testscftargetdim (test.data.testGeoOpt.ADFGeoOptTest)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 144, in testscftargetdim
<     dim_scfvalues = (len(self.data.scfvalues),len(self.data.scfvalues[0][0]))
< AttributeError: 'ccData' object has no attribute 'scfvalues'
< 
< Error for testscfvaluetype (test.data.testGeoOpt.ADFGeoOptTest)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/testGeoOpt.py", line 120, in testscfvaluetype
<     self.assertEqual(type(self.data.scfvalues),type([]))
< AttributeError: 'ccData' object has no attribute 'scfvalues'
< 
---
>   /home/eric/development/cclib_berquist/test/../data/regression/ADF/ADF2004.01/dvb_gopt.adfout ... parsed and tested
43,133c13,16
<   /home/eric/development/cclib_berquist/test/../data/regression/ADF/ADF2004.01/dvb_sp.adfout ... 3 test(s) had errors
< Error for testaooverlaps (test.data.testSP.ADFSPTest)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   [Previous line repeated 3 more times]
<   File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 219, in testaooverlaps
<     self.assertEqual(self.data.aooverlaps.shape, (self.data.nbasis, self.data.nbasis))
< AttributeError: 'ccData' object has no attribute 'aooverlaps'
< 
< Error for testscftargetdim (test.data.testSP.ADFSPTest)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 163, in testscftargetdim
<     self.assertEqual(self.data.scftargets.shape, (len(self.data.scfvalues), len(self.data.scfvalues[0][0])))
< AttributeError: 'ccData' object has no attribute 'scfvalues'
< 
< Error for testscfvaluetype (test.data.testSP.ADFSPTest)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 154, in testscfvaluetype
<     self.assertEqual(type(self.data.scfvalues),type([]))
< AttributeError: 'ccData' object has no attribute 'scfvalues'
< 
<   /home/eric/development/cclib_berquist/test/../data/regression/ADF/ADF2004.01/dvb_sp_b.adfout ... 2 test(s) had errors
< Error for testscftargetdim (test.data.testSP.ADFSPTest)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 163, in testscftargetdim
<     self.assertEqual(self.data.scftargets.shape, (len(self.data.scfvalues), len(self.data.scfvalues[0][0])))
< AttributeError: 'ccData' object has no attribute 'scfvalues'
< 
< Error for testscfvaluetype (test.data.testSP.ADFSPTest)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 154, in testscfvaluetype
<     self.assertEqual(type(self.data.scfvalues),type([]))
< AttributeError: 'ccData' object has no attribute 'scfvalues'
< 
<   /home/eric/development/cclib_berquist/test/../data/regression/ADF/ADF2004.01/dvb_sp_c.adfout ... 3 test(s) had errors
< Error for testaooverlaps (__main__.ADFSPTest_nosyms_valence)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   [Previous line repeated 3 more times]
<   File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 219, in testaooverlaps
<     self.assertEqual(self.data.aooverlaps.shape, (self.data.nbasis, self.data.nbasis))
< AttributeError: 'ccData' object has no attribute 'aooverlaps'
< 
< Error for testscftargetdim (__main__.ADFSPTest_nosyms_valence)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 163, in testscftargetdim
<     self.assertEqual(self.data.scftargets.shape, (len(self.data.scfvalues), len(self.data.scfvalues[0][0])))
< AttributeError: 'ccData' object has no attribute 'scfvalues'
< 
< Error for testscfvaluetype (__main__.ADFSPTest_nosyms_valence)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 154, in testscfvaluetype
<     self.assertEqual(type(self.data.scfvalues),type([]))
< AttributeError: 'ccData' object has no attribute 'scfvalues'
< 
<   /home/eric/development/cclib_berquist/test/../data/regression/ADF/ADF2004.01/dvb_sp_d.adfout ... 3 test(s) had errors
< Error for testaooverlaps (__main__.ADFSPTest_nosyms)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   File "/home/eric/development/cclib_berquist/test/data/skip.py", line 18, in testwrapper
<     testfunc(self, *args, **kwargs)
<   [Previous line repeated 3 more times]
<   File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 219, in testaooverlaps
<     self.assertEqual(self.data.aooverlaps.shape, (self.data.nbasis, self.data.nbasis))
< AttributeError: 'ccData' object has no attribute 'aooverlaps'
< 
< Error for testscftargetdim (__main__.ADFSPTest_nosyms)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 163, in testscftargetdim
<     self.assertEqual(self.data.scftargets.shape, (len(self.data.scfvalues), len(self.data.scfvalues[0][0])))
< AttributeError: 'ccData' object has no attribute 'scfvalues'
< 
< Error for testscfvaluetype (__main__.ADFSPTest_nosyms)
< Traceback (most recent call last):
<   File "/home/eric/development/cclib_berquist/test/data/testSP.py", line 154, in testscfvaluetype
<     self.assertEqual(type(self.data.scfvalues),type([]))
< AttributeError: 'ccData' object has no attribute 'scfvalues'
< 
---
>   /home/eric/development/cclib_berquist/test/../data/regression/ADF/ADF2004.01/dvb_sp.adfout ... parsed and tested
>   /home/eric/development/cclib_berquist/test/../data/regression/ADF/ADF2004.01/dvb_sp_b.adfout ... parsed and tested
>   /home/eric/development/cclib_berquist/test/../data/regression/ADF/ADF2004.01/dvb_sp_c.adfout ... parsed and tested
>   /home/eric/development/cclib_berquist/test/../data/regression/ADF/ADF2004.01/dvb_sp_d.adfout ... parsed and tested
825c708
< Total: 437   Failed: 0  Errors: 35
---
> Total: 437   Failed: 0  Errors: 21
```

File: test/regression.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def testADF_ADF2013_01_stopiter_dvb_sp_b_adfout(logfile):
 
 def testADF_ADF2013_01_stopiter_dvb_sp_c_adfout(logfile):
     """This logfile has not SCF test lines so we have no way to check what happens."""
-    # This is what we would have checked:
+    # This is what we ADFwould have checked:
```
Comment (berquist) [inline test/regression.py]: Something weird happened here...

Comment (amandadumi) [reply test/regression.py]: Apologies, this typo is now addressed. Thank you for pointing out the git config oddity. I hopefully have addressed that as well, but let me know!

File: test/regression.py [hunk 2]
```diff
@@ -1810,12 +1823,40 @@ class ADFSPTest_nosyms(ADFSPTest, GenericSPTest):
...
+        """SCF cycles were not printed here."""
+
+    @unittest.skip('Cannot parse scfvalues from this file.')
+    def testscfvaluetype(self):
+        """SCF cycles were not printed here."""
+
+    @unittest.skip('Cannot parse aooverlaps from this file.')
+    def testaooverlaps(self):
+        """AO overlaps were not printed here."""
+
```
Comment (langner) [inline test/regression.py]: Let's follow PEP8 and surround class definitions with two blank lines: https://www.python.org/dev/peps/pep-0008/#blank-lines

Comment (amandadumi) [reply test/regression.py]: No problem! Thank you for the link. I have made these changes for the classes I've introduced into this file. Would you like me to change this for all classes in the regression.py file?

File: test/regression.py (hunk 1 above)
Comment (langner) [reply test/regression.py]: Sure, if you're up for it.

File: test/regression.py (hunk 2 above)
Comment (langner) [reply test/regression.py]: Sure, if you're up for it.

**Comment by langner** [comment]
Something weird also happened with commit bea0098... did you rebase my commit? You should be rebasing you own commits, so that the history doesn't get garbled.

**Comment by amandadumi** [comment]
Im honestly not sure what happened there or how I failed to notice before pushing. I realized I messed up quickly, but wasn't sure how to correct it at first. It seems to be resolved now.

 I can definitely fix the rest of the regression file. Should I make that as a separate pull request?

**Comment by langner** [comment]
> I can definitely fix the rest of the regression file. Should I make that as a separate pull request?

Sure, you can do it separately.

Thanks for the work!

**Comment by berquist** [comment]
I didn't get to it in time to mention it, but had hope for this to be merged into the `673-fix-silent-regression-failures` rather than `master` as in https://github.com/cclib/cclib/pull/675 so we could track progress.

**Comment by langner** [comment]
Oh, sorry about that! We could revert and do it right, but probably not worth it.

**Pull request merged**